Source/WebCore/rendering/FloatingObjects.h

```diff
@@ -53,6 +53,7 @@ class FloatingObject {
 
     Type type() const { return static_cast<Type>(m_type); }
     RenderBox& renderer() const { ASSERT(m_renderer); return *m_renderer; }
+    bool hasValidRenderer() const { return m_renderer.get() != nullptr; }
```

Comment:
I think it'd be better to figure out why we end up with invalid renderer here as that may even fix an actual rendering glitch.

Reply:
Yeah, I don't think we should be adding this check. Having `FloatingObject` without the corresponding `RenderObject` is a messed up state.